nikola/plugins/compile/rest/post_list.py

```diff
@@ -144,20 +149,22 @@ def run(self):
         categories = [c.strip().lower() for c in categories.split(',')] if categories else []
         slugs = self.options.get('slugs')
         slugs = [s.strip() for s in slugs.split(',')] if slugs else []
+        post_type = self.options.get('post_type')
         show_all = self.options.get('all', False)
         lang = self.options.get('lang', utils.LocaleBorg().current_lang)
         template = self.options.get('template', 'post_list_directive.tmpl')
         sort = self.options.get('sort')
 
-        output = _do_post_list(start, stop, reverse, tags, categories, slugs, show_all,
-                               lang, template, sort, state=self.state, site=self.site)
+        output = _do_post_list(start, stop, reverse, tags, categories, slugs, post_type,
+                    show_all, lang, template, sort, state=self.state, site=self.site)
```

Comment:
Fix the indentation so that you follow PEP 8 (`show_all` directly under `start`)